gulpfile.js

```diff
@@ -0,0 +1,46 @@
+var gulp = require("gulp");
+var watch = require('gulp-watch');
+var plumber = require('gulp-plumber');
+var webpack = require('webpack-stream');
+var named = require('vinyl-named');
+var browserSync = require('browser-sync').create();
+var babel = require('gulp-babel');
+var nodemon = require('gulp-nodemon');
+var config = require('./webpack.config.js');
+
+gulp.task("compile", function() {
+    var stream =
+        gulp.src(['src/index.jsx'])
+        .pipe(named())
+        .pipe(plumber())
+        .pipe(webpack( config ))
+        .pipe(gulp.dest("build/src"));
+    return stream;
+})
+
+gulp.task('BSRefresh', function (done) {
```

Comment:
Is this duplicated by the compile-watch task or is it just useful for something else?

Reply:
@chris-bacon Good spot, this is duplicate code!

Reply:
Pushed to remove redundant code